### PR TITLE
Update handbook-check-failures to add use case when reports_to slug is invalid because of missing manager_role_slug.md

### PR DESCRIPTION
**Pull request by Tobiryd**

### CODE REVIEW

File: content/handbook/editing/handbook-check-failures.md [hunk 1]
```diff
@@ -123,6 +123,15 @@ YAMLException: bad indentation of a mapping entry (19:21)
  21 |   role: 'VP Product'
 ```
 
+### YAML Data Validation Errors on 'reports_to' slug
+
+If you get an error on the 'reports_to' slug, it is likely that your manager hasn't declared their 'manager_role_slug'.
```
Comment (kelsey-brown) [inline content/handbook/editing/handbook-check-failures.md]:
```suggestion
If you get an error on the 'reports_to' slug when adding yourself to the Team page, it is likely that your manager hasn't declared their 'manager_role_slug'.
```

**Review by kelsey-brown — APPROVED** [review]
Thank you for documenting this! Made one suggestion to clarify this happens when adding yourself to the team page (I'm assuming that's the case?)

File: content/handbook/editing/handbook-check-failures.md (hunk 1 above)
Comment (Tobiryd) [reply content/handbook/editing/handbook-check-failures.md]: Yes, that's better. Thanks!

**Review by jhchabran — COMMENTED** [review]
Thank you!

**Pull request merged**